doc/field_guide__rr_lyrae.md

```diff
@@ -0,0 +1,47 @@
+## RR Lyrae (rrlyr)
+RR Lyrae are pulsating horizontal branch dA/dF stars with a mass 0.5 of Solar masses.
+They are part of the main 'instability strip' in the HR-diagram,
+where stars pulsate due to the double ionisation of He.
+The two main subtypes are `ab` and `c`,
+
+### Classification and numbers
+- Supertypes
+  - variable
+  - periodic
+  - pulsator
+- Subtypes
+  - RR Lyrae ab
+  - RR Lyrae c
+- Occurrence rate: very common, about 10<sup>5</sup> expected in ZTF data
+
+
+### ZTF light curves
+![ZTF rrlyr ab](data/rr_lyr_ab.png)
+![ZTF rrlyr c](data/rr_lyr_c.png)
+
+#### Description
+RR Lyrae (ab) are easy to recognise by their distinctive light curve shape and
+high amplitude variability.
+RR Lyrae (c) are a bit more of a challenge since their light curve shape is more sinusoidal.
+For low-SNR cases they can sometimes be confused with contact binaries (EW),

```

Comment:
Giving the "can be confused with" information is very useful. Any way there can be links to examples of the two types side-by-side?